borg/archiver.py

```diff
@@ -1076,6 +1107,34 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='REPOSITORY', nargs='?', default='',
                                type=location_validator(archive=False))
 
+        subparser = subparsers.add_parser('key-export', parents=[common_parser],
```

Comment:
Where they are declared is not overly important I think (although adding it directly into keymanager may make maintenance easier in the future).

The part about having a `borg key` subcommand which keymanager-related commands is something we all agree on, yes?